documentation/ntfs/ntfs_fare_extention.md

```diff
@@ -0,0 +1,59 @@
+NTFS - Fare extension
+======================================
+
+# Introduction
+This document describe an extension of the core [NTFS (fr)](./ntfs_fr.md) data format to add fares and tickets informations into Navitia.
+The [old fare extension](./ntfs_fare_extension_fr_deprecated.md) is still in use in conjunction with the files described here for retrocompability matters.
+
+# Data format
+The data format of the files are the same as the core (NTFS ones](./ntfs_fr.md). In short, files are CSV encoded in utf8 and with a comma `,` separator.
+
+# Files list of this extension
+
+File | Constraint | Comment
+--- | --- | ---
+tickets.txt | Required | Contains tickets with prices.
+ticket_uses.txt | Required | Contains where the ticket can be used
+ticket_use_perimeters.txt | Required | Defines the networks and lines of a ticket_use
+ticket_use_restrictions.txt | Optionnal | Defines a use of a ticket between zones or stop_areas
+
+# Files descriptions
+
+## tickets.txt (Required)
+Field | Type | Constraint | Description
+--- | --- | --- | ---
+ticket_id | Text | Required | Unique ID of the ticket
+ticket_name | Text | Required | Name of the ticket (may be displayed to passengers)
+ticket_price | Non-negative Float | Required |
+ticker_currency | Currency Code | Required | An ISO 4217 alphabetical currency code. For the list of current currency, refer to https://en.wikipedia.org/wiki/ISO_4217#Active_codes.
+ticket_comment | Text | Optionnal | Comment on the ticket that may be displayed to the passenger
+ticket_validity_start | Date | Required | First day of usability of the ticket
+ticket_validity_end | Date | Required | Last day of usability of the ticket
+
+## ticket_uses.txt (Required)
+Field | Type | Constraint | Description
+--- | --- | --- | ---
+ticket_id | Text | Required | Unique ID of the ticket
+ticket_use_id | Text | Required | Unique ID of the ticket use
```

Comment:
I would have put the `ticket_use_id` in `ticket.txt` to allow multiple ticket to use the same set of rules defined by a ticket_used. This way it will be easier to handle change of price

Reply:
yes, good point ! It also enable having a ticket without a known price. I'll make a change in this way